OrbitCore/CaptureData.h

```diff
@@ -8,6 +8,7 @@
 #include <memory>
 #include <vector>
 
+#include "../../OrbitGl/TracepointCustom.h"
```

Comment:
I left a similar comment below: OrbitCore must not depend on OrbitGl. Consider moving TracepointCustom.h to a shared place.

Reply:
Yes, this is right. I will move it to make the includes more visible.